widgets/nuxeo-user-suggestion.html

```diff
@@ -245,6 +245,9 @@
           if (item) {
             switch (item['entity-type']) {
               case 'user':
+                if (item.properties && item.properties.firstName && item.properties.lastName) {
+                  return `${item.properties.firstName} ${item.properties.lastName}`;
+                }
                 return item.id;
```

Comment:
Should we check if item.id exists?
Also, on default we are also doing that logics... should we rely on that and keep only the recently inserted code?

Reply:
if we have `item['entity-type']` but no `id` I believe there is an issue somewhere in Rest API level. Plus I wouldn't know what to use ..